### PR TITLE
Update 13/STORE: new error codes for Extended HistoryResponse

### PR DESCRIPTION
**Pull request by NagyZoltanPeter**
This is a small adjustment on new error codes introduced for waku's store protocol response.
Change is introduced by https://github.com/waku-org/pm/issues/66 implementation.

### CODE REVIEW

**Review by jimstir — APPROVED** [review]

**Review by jm-clius — COMMENTED** [review]
I apologise for confusion with new and existing RFCs and different repos, but indeed we are replacing this RFC 13 with a new version of Store (v3) where we do not explicitly define the error codes (exactly because we expect the list to grow significantly and do not want to have to update the RFC every time). Store v3 spec however still only exists in a branch as we have been modifying it extensively during the POC implementation. However, I think we are close to merging it into waku.org/specs as a new spec (but not RFC).

TL;DR - we not need to update this spec as we're moving to Store v3: https://github.com/vacp2p/rfc/pull/665/files

**Comment by jimstir** [comment]
> I apologise for confusion with new and existing RFCs and different repos, but indeed we are replacing this RFC 13 with a new version of Store (v3) where we do not explicitly define the error codes (exactly because we expect the list to grow significantly and do not want to have to update the RFC every time). Store v3 spec however still only exists in a branch as we have been modifying it extensively during the POC implementation. However, I think we are close to merging it into waku.org/specs as a new spec (but not RFC).
> 
> TL;DR - we not need to update this spec as we're moving to Store v3: https://github.com/vacp2p/rfc/pull/665/files

@NagyZoltanPeter I will close this pr, when Store v3 is ready it will live in waku/specs repo.

**Comment by NagyZoltanPeter** [comment]
> > I apologise for confusion with new and existing RFCs and different repos, but indeed we are replacing this RFC 13 with a new version of Store (v3) where we do not explicitly define the error codes (exactly because we expect the list to grow significantly and do not want to have to update the RFC every time). Store v3 spec however still only exists in a branch as we have been modifying it extensively during the POC implementation. However, I think we are close to merging it into waku.org/specs as a new spec (but not RFC).
> > TL;DR - we not need to update this spec as we're moving to Store v3: https://github.com/vacp2p/rfc/pull/665/files
> 
> @NagyZoltanPeter I will close this pr, when Store v3 is ready it will live in waku/specs repo.

Yes, its ok! Thanks for the heads up.
